common4j/src/main/com/microsoft/identity/common/java/providers/microsoft/MicrosoftTokenRequest.java

```diff
@@ -106,6 +106,12 @@ public MicrosoftTokenRequest() {
     @Accessors(prefix = "m")
     private boolean mPKeyAuthHeaderAllowed;
 
+    // Send Passkey Auth header for interactive calls with WebView as authorization agent.
+    @Getter
+    @Setter
+    @Accessors(prefix = "m")
+    private boolean mPasskeyAuthHeaderAllowed;
```

Comment:
You want this being sent to eSTS in https request body? Probably not. If not then you should mark this as `transient`

Reply:
Removed these lines as a result of one of the previous comments above.